### ISSUE TITLE
can we make a website more powerful?

### ISSUE THREAD

**Issue by hunterhug** [TTyb/tbtmGoods#1]
It sounds great to make a Sass which data can keep in a remore machine.

**Comment by TTyb** [comment]
I agree to develop website by flask if time permits,but a remore machine is expensive .
making money by website  ?

**Comment by hunterhug** [comment]
I have one, But the data must crontab to clear for I have only 30G store
